examples/scripts/post_install.bat

```diff
@@ -3,6 +3,7 @@ if not "%INSTALLER_NAME%" == "Scripts" exit 1
 if not "%INSTALLER_VER%" == "X" exit 1
 if not "%INSTALLER_PLAT%" == "win-64" exit 1
 if not "%INSTALLER_TYPE%" == "EXE" exit 1
+if "%INSTALLER_UNATTENDED%" == "" exit 1
```

Comment:
Since the tests are run in batch/headless/silent mode, should we assert that the value be 1?

Reply:
Let's try :)

```suggestion
if not "%INSTALLER_UNATTENDED%" == "1" exit 1
```

Reply:
Let's expand this to the other bat file and the sh scripts (`[[ "${INSTALLER_UNATTENDED}" == "1" ]] || exit 1` or something) and I think we have a good test here.

Reply:
Oops you are right, I forgot the other files. Added some additional tests.

Reply:
Seems to be passing.